avocado_vt/plugins/vt_joblock.py

```diff
@@ -123,8 +106,9 @@ def pre(self, job):
                 self._lock(job)
         except Exception as detail:
             msg = "Failure trying to set Avocado-VT job lock: %s" % detail
-            self._abort(msg, job)
+            self.log.error(msg)
+            sys.exit(exit_codes.AVOCADO_JOB_FAIL | job.exitcode)
 
     def post(self, job):
         if self.lock_file is not None:
-            self._unlock()
+            os.unlink(self.lock_file)
```

Comment:
Well currently interruption inside `post` interrupts everything. We might introduce something like `atexit` does, which means ignore the exceptions and proceed with next function...